xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/action-menus.less

```
@@ -149,6 +149,10 @@
       width: 0;
     }
   }
+
+  div.suggestItems .hide-button{
```

```suggestion
  div.suggestItems .hide-button {
```

Also, I'm not fully sure about the location. I mean, I understand the need for LESS, but I'd put this rule maybe in ``misc.less`` since it's not related to action menus (the same search suggest can be used in many other places).